components/description-list/test/description-list.visual-diff.js

```diff
@@ -0,0 +1,44 @@
+import puppeteer from 'puppeteer';
+import VisualDiff from '@brightspace-ui/visual-diff';
+
+describe('d2l-dl-wrapper', () => {
+
+	const visualDiff = new VisualDiff('description-list', import.meta.url);
+
+	let browser, page;
+
+	before(async() => {
+		browser = await puppeteer.launch();
+		page = await visualDiff.createPage(browser);
+		await page.goto(`${visualDiff.getBaseUrl()}/components/description-list/test/description-list.visual-diff.html`, { waitUntil: ['networkidle0', 'load'] });
+		await page.bringToFront();
+	});
+
+	after(async() => await browser.close());
+
+	[
+		'default',
+		'long',
+		'activity-display',
+		'bulk-course-import',
+		'slotted',
+	].forEach((name) => {
+		const selector = `#${name}`;
+		[
+			799,
+			599,
+			299,
+			199,
```

Comment:
Since the default breakpoint to stack is <200 at the moment, need to make the test width 199 to show the stacked width.